paper/paper.md

```diff
@@ -0,0 +1,116 @@
+---
+title: 'pyOBIS: easy access to taxonomic occurrence records harvested from thousands of datasets'
+tags:
+  - Python
+  - oceanography
+  - marine data
+authors:
+  - name: Scott Chamberlain
+    equal-contrib: true
+  - name: Ayush Anand
+    equal-contrib: true
+    affiliation: 1
+  - name: Tylar Murray
+    corresponding: true
+    affiliation: 2
+  - name: Filipe Fernandes
+    corresponding: true
+  - name: Mathew Biddle
+    corresponding: true
+    affiliation: 3
+affiliations:
+ - name: National Institute of Technology Durgapur, India
+   index: 1
+ - name: IMaRS University of South Florida, US
+   index: 2
+ - name: National Oceanic and Atmospheric Administration, National Ocean Service, Integrated Ocean Observing System, US
+   index: 3
+date: 9 May 2023
+bibliography: paper.bib
+---
+
+# Summary
+The pyOBIS python package provides easy access to taxonomic occurrence records harvested from thousands of datasets.
+The package uses the API from the Ocean Biodiversity Information System (OBIS),
+a global open-access data and information clearinghouse on marine data for biodiversity for science, conservation,
+and sustainable development.
+OBIS has more than 107 million occurrence records, making availibility of ocean data possible but accesibility remains a challenge.
+pyOBIS solves the challenge by providing built-in functions for accessing data on occurrences, taxons, nodes, checklists, and dataset metadatas.
+Users can download, visualize, segment, process and export data to any format of your choice with its built-in tools or rich ecosystem of libraries in python.
+Coupled together with other libraries like [pyDwcViz](https://github.com/marinebon/py-dwc-viz),
+it forms an ecosystem of analysing Darwin Core Data with super ease through built-in functions.
+
+# Introduction
+OBIS is a global open-access data and information warehouse on marine biodiversity data.
+It contains occurrence records, dataset metadatas, environmental data around species occurrences,
+and many more biogeographic pointers.
+The package provides easy export of data to Pandas DataFrame to help researchers focus more on analysis rather than data mining,
+and several included Jupyter notebooks demonstrate example analyses that can be used as a starting point for addressing research questions related to global and local distributions of species across space and time.
+
+
+# Why pyOBIS?
+pyOBIS is intuitively split into different modules for querying IUCN red lists,
+newly added species, datasets added, information on OBIS nodes, occurrence records,
+MeasurementOrFacts, eDNA records, etc and searchable through unique IDs, taxa, scientific names,
+geolocation, timestamps, and others.
+The Taxa IDs used by OBIS is adopted from annotations by the WoRMS team thereby maintaining a uniform and universal identification convention.
+
+## Main Features
```

Comment:
I have also added some more features highlighting why it is not just an API wrapper. @MathewBiddle if you would like to add something out of experience using this package.